app/views/certificates.html

```diff
@@ -0,0 +1,209 @@
+<div class="page-content">
+  <div class="row">
+    <!-- include the sidebar -->
+    <div ng-include="'views/sidebar.html'"></div>
+    <div class="col-md-10">
+      <div class="content-box-large">
+        <div class="panel-heading">
+          <div class="panel-title">
+            <h2><i class="glyphicon glyphicon-certificate"></i>&nbsp;&nbsp;Certificates</h2>
+          </div>
+        </div>
+        
+        <div class="exportSectionDiv">
+          <div style="float: left; width: 100%">
+            <h3><strong>Server Certificate</strong></h3>
+
+            <table class="table table-striped transactionTable">
+              <tbody>
+                <tr>
+                  <td width="120">Comman Name: </td>
+                  <td data-title="Comman Name">{{currentServerCert.commonName}}</td>
+                </tr>
+                <tr>
+                  <td>Organization</td>
+                  <td data-title="Organization">{{currentServerCert.organization}}</td>
+                </tr>
+                <tr>
+                  <td>Organization Unit</td>
+                  <td data-title="Organization Unit">{{currentServerCert.organizationUnit}}</td>
+                </tr>
+                <tr>
+                  <td>Country</td>
+                  <td data-title="Country">{{currentServerCert.country}}</td>
+                </tr>
+                <tr>
+                  <td>State</td>
+                  <td data-title="State">{{currentServerCert.state}}</td>
+                </tr>
+                <tr>
+                  <td>Locality</td>
+                  <td data-title="Locality">{{currentServerCert.locality}}</td>
+                </tr>
+                <tr>
+                  <td>Email Address</td>
+                  <td data-title="Email Address">{{currentServerCert.emailAddress}}</td>
+                </tr>
+                <tr>
+                  <td>Validity</td>
+                  <td data-title="Validity">
+                    Start: <strong>{{ currentServerCert.validity.start | date:'yyyy-MM-dd HH:mm:ss Z' }}</strong><br />
+                    End: <strong>{{ currentServerCert.validity.end | date:'yyyy-MM-dd HH:mm:ss Z' }}</strong></td>
+                </tr>
+                <tr>
+                  <td>Certificate</td>
+                  <td data-title="Certificate">{{currentServerCert.data}}</td>
```

Comment:
Perhaps we should make this a read-only textarea so that the certificate's newline are respected. Also, then then we don't have to show the whole thing, they can scroll to see the rest.